lib/generators/v3/index.js

```diff
@@ -0,0 +1,239 @@
+'use strict';
+const Chance = require('chance').Chance();
+const Format = require('./format');
+const Randexp = require('randexp').randexp;
+
+const mock = (node, useExample)  => {
```

Comment:
Notice I'm calling this node instead of schema, as the schema is actually `node.schema` now. 

![screen shot 2018-07-24 at 10 57 43 am](https://user-images.githubusercontent.com/67381/43147176-6e9f6c9a-8f30-11e8-94e1-46428d053f59.png)